ui/tank_rogue/presets.ts

```diff
@@ -185,6 +185,23 @@ export const DefaultTalentsSubtlety = TalentPresets[Phase.Phase5][0];
 
 export const DefaultTalents = DefaultTalentsCombat;
 
+///////////////////////////////////////////////////////////////////////////
+//                                Encounters
+///////////////////////////////////////////////////////////////////////////
+export const PresetBuildEncounterDefault = PresetUtils.makePresetBuild('Default', {
```

Comment:
@sanguinerarogue I added these as an example, please feel free to modify these in this branch!